standalone/DimensionReduction/main.cpp

```diff
@@ -0,0 +1,437 @@
+/// \author Alexandre Talon <alexandre.talon@lip6.fr>.
+/// \date November 2023
+///
+/// \brief dummy program example.
+
+// TTK Includes
+#include <CommandLineParser.h>
+#include <ttkDimensionReduction.h>
+
+// VTK Includes
+#include <vtkCellData.h>
+#include <vtkDataObject.h>
+#include <vtkDataSet.h>
+#include <vtkDelimitedTextReader.h>
+#include <vtkDelimitedTextWriter.h>
+#include <vtkPointData.h>
+#include <vtkSmartPointer.h>
+#include <vtkTable.h>
+
+int main(int argc, char **argv) {
+  std::vector<std::string> inputFilePaths;
+  std::vector<std::string> inputArrayNames;
+  std::string outputArrayName{"2d-projection"};
+  std::string outputPathPrefix{"output"};
+  bool listArrays{false};
+
+  // Variables for the module options.
+  int backend{0};
```

Comment:
I'd argue to use a different default value. 0 is for Spectral Embedding. I guess MDS would be more standard (check the defaults in the ParaView *xml file).

Reply:
Fixed, MDS it is indead.